cypress/e2e/inventory/cataloging/maintaining-the-catalog/add-item-to-an-existing-title.cy.js

```diff
@@ -0,0 +1,85 @@
+import uuid from 'uuid';
+
+import { DevTeams, TestTypes, Permissions } from '../../../../support/dictionary';
+import { Locations, ServicePoints } from '../../../../support/fragments/settings/tenant';
+import { MATERIAL_TYPE_NAMES, LOAN_TYPE_NAMES } from '../../../../support/constants';
+import TopMenu from '../../../../support/fragments/topMenu';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import InventoryInstances from '../../../../support/fragments/inventory/inventoryInstances';
+import InventoryHoldings from '../../../../support/fragments/inventory/holdings/inventoryHoldings';
+import InventorySearchAndFilter from '../../../../support/fragments/inventory/inventorySearchAndFilter';
+import Users from '../../../../support/fragments/users/users';
+
+describe('Inventory', () => {
+  describe('Cataloging -> Maintaining the catalog', () => {
+    const testData = {
+      barcode: uuid(),
+      servicePoint: ServicePoints.getDefaultServicePoint(),
+      instance: {},
+      holding: {},
+    };
+
+    before('Create test data', () => {
+      cy.getAdminToken().then(() => {
+        InventoryInstance.createInstanceViaApi().then(({ instanceData }) => {
+          testData.instance = instanceData;
+
+          testData.location = Locations.getDefaultLocation({
+            servicePointId: testData.servicePoint.id,
+          }).location;
+
+          Locations.createViaApi(testData.location).then((location) => {
+            InventoryHoldings.createHoldingRecordViaApi({
+              instanceId: testData.instance.instanceId,
+              permanentLocationId: location.id,
+            }).then((holding) => {
+              testData.holding = holding;
+            });
+          });
+        });
+      });
+
+      cy.createTempUser([Permissions.inventoryAll.gui]).then((userProperties) => {
+        testData.user = userProperties;
+
+        cy.login(testData.user.username, testData.user.password, {
+          path: TopMenu.inventoryPath,
+          waiter: InventoryInstances.waitContentLoading,
+        });
+      });
+    });
+
+    after('Delete test data', () => {
+      InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(testData.barcode);
+      Users.deleteViaApi(testData.user.userId);
+    });
```

Comment:
could you please also delete the location

Reply:
Done